lib/screens/department_screen/department_screen.dart

```diff
@@ -111,7 +111,10 @@ class _DepartmentScreenState extends State<DepartmentScreen> {
 
   @override
   void dispose() {
-    overlayEntry!.remove();
+    if (overlayEntry != null) {
+      overlayEntry!.remove();
+    }
+
     overlayEntry = null;
```

Comment:
No need to set it to null

Reply:
if it isn't set to null, then say I click the overlay thing multiple times, multiple instances get created and the if condition also gets passed

Reply:
Cool